Jenkinsfile

```diff
@@ -45,11 +45,11 @@
 
 // NOTE: these lines are scanned by docker/dev_common.sh. Please update the regex as needed. -->
 ci_lint = "tlcpack/ci-lint:v0.62"
-ci_gpu = "tlcpack/ci-gpu:v0.70"
-ci_cpu = "tlcpack/ci-cpu:v0.70"
-ci_wasm = "tlcpack/ci-wasm:v0.70"
-ci_i386 = "tlcpack/ci-i386:v0.70"
-ci_qemu = "tlcpack/ci-qemu:v0.01"
+ci_gpu = "tlcpack/ci-gpu:v0.71"
+ci_cpu = "tlcpack/ci-cpu:v0.71"
```

Comment:
please do not change these flags, and keep them as they are, we will need to first merge in the python package change, and make sure the test pass